src/Grid/Util/Trait/MetaData/LocalizedValueTrait.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Grid\Util\Trait\MetaData;
```

Comment:
Since this is only for Assets. We should move it into a Asset Namespace? 
And add a Check if $element is an instance of Asset?

Reply:
I would not move it to the asset namespace since it is clearly only a grid functionality, with the column parameter. 
I already moved it to the subnamespace metadata but of course we could  rename it to AssetMetaData. I will add the check, good catch.